controllers/flinkcluster_reconciler.go

```diff
@@ -575,14 +573,15 @@ func (reconciler *ClusterReconciler) getFlinkJobID() string {
 	return ""
 }
 
-func (reconciler *ClusterReconciler) restartJob() error {
+func (reconciler *ClusterReconciler) restartJob(shouldTakeSavepoint bool) error {
```

Comment:
@shashken @functicons `restartJob` is just for restarting the job so there is no need to trigger savepoint here.